src/main/java/frc/robot/RobotMap.java

```diff
@@ -31,4 +31,8 @@ public class RobotMap {
   public static final int INTAKE_PIVOT_MOTOR_CAN_ID = Constants.PLACEHOLDER_INT,
       INTAKING_FRONT_MOTOR_CAN_ID = Constants.PLACEHOLDER_INT,
       INTAKING_BACK_MOTOR_CAN_ID = Constants.PLACEHOLDER_INT;
+
+    /** CANdle for the lights */
```

Comment:
this is nitpicky but the comment is kind of misleading, in my opinion. either delete it or edit